python/lsst/ts/envsensors/device/base_device.py

```diff
@@ -112,16 +107,19 @@ async def _run(self) -> None:
         while not self._telemetry_loop.done():
             self._log.debug("Reading data.")
             tm: float = time.time()
+            # TODO: Catch the exceptions and assign the corresponding
+            #  ResponseCode.
```

Comment:
Please file a ticket and reference it here (for all TODOs).

Reply:
Regarding line 109: `tm: float = time.time()`: please use TAI if possible. If you can import salobj here then `tm = salobj.tai()`.

Also please consider a clearer variable name (especially as some operating systems helpfully try to convert tm to the trademark symbol), such as `tai` or `curr_tai` or if you really need utc...

Reply:
Ticket added to the TODO and switched to using salobj.